### PR TITLE
[Collecticiel] - valuation comments note visible

### PR DESCRIPTION
**Pull request by pitrackster**
resolve https://github.com/claroline/Distribution/issues/1059

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

File: plugin/collecticiel/Resources/views/Document/viewNotation.html.twig [hunk 1]
```diff
@@ -24,18 +24,19 @@
             <!-- #MODAL BODY -->
             <div class="modal-body">
 
-                <div class="row">
-                    <div class="col-md-12">
-                        {# Bloc "Rappel des instructions" #}
-                        {% if dropzone.instruction != null %}
+            <div class="modal-body">
```
Comment (Elorfin) [inline plugin/collecticiel/Resources/views/Document/viewNotation.html.twig]: This tag is duplicated.

Comment (pitrackster) [inline plugin/collecticiel/Resources/views/Document/viewNotation.html.twig]: @Elorfin ... thanks...

**Comment by Elorfin** [comment]
LGTM

**Pull request merged**